docs/public/css/style.css

```diff
@@ -0,0 +1,99 @@
+main {
+  padding: 1rem;
+
+  & * {
+    scroll-margin-top: 4rem;
+  }
+
+  @media print {
+    width: 100%!important;
+    --bs-code-color: --bs-body-color;
+  }
+}
+
+header {
+  z-index: 1050!important; /* higher than #contents */
+}
+
+section[id] {
+  display: flow-root;
+
+  > :first-child {
+    .icon-link {
+      opacity: 0;
+
+      &:focus {
+        opacity: .8;
+      }
+    }
+
+    &:hover .icon-link {
+      opacity: 1;
+    }
+  }
+}
+
+.offcanvas[aria-modal=true] .nav-link {
+  padding: .7rem;
+}
+.offcanvas-md {
+  --bs-offcanvas-width: auto;
+}
+
+ul {
+  list-style: none;
+  padding-left: 1rem;
+}
+
+pre {
+  margin: 0 .5rem 1rem;
+  padding: .5rem 1rem;
+  background-color: var(--bs-secondary-bg-subtle);
+  border: var(--bs-border-width) var(--bs-border-style) var(--bs-border-color);
+}
+
+button{
+  &[aria-expanded=false] .bi-chevron-down {
+    display: none;
+  }
```

Comment:
Wow, this is cutting edge technology! My Firefox doesn't even [support](https://caniuse.com/css-nesting) this by default yet.

Reply:
Yeah, I feel sorry for Firefox users but I know that. The feature will be turned on soon; at the end of this month.